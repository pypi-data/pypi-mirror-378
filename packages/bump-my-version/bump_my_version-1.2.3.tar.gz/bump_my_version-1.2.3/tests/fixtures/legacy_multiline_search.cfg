[bumpversion]
current_version = 1.0.0

[bumpversion:file:MULTILINE_SEARCH.md]
search = **unreleased**
	**v{current_version}**
replace = **unreleased**
	**v{new_version}**
