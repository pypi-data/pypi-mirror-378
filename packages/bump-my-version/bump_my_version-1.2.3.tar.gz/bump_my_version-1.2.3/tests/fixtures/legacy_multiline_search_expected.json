{
  "current_version": "1.0.0",
  "files": [
    {
      "filename": "MULTILINE_SEARCH.md",
      "search": "**unreleased**\n**v{current_version}**",
      "replace": "**unreleased**\n**v{new_version}**"
    }
  ],
  "parts": {}
}
